CMakeLists.txt

```diff
@@ -305,10 +305,10 @@ else()
 endif()
 
 if(EMSCRIPTEN)
-  if("${CMAKE_BUILD_TYPE}" MATCHES "Debug")
-    add_link_flag("-sERROR_ON_WASM_CHANGES_AFTER_LINK")
-    add_link_flag("-sWASM_BIGINT")
-  else()
+  # Note: to debug with DWARF you will need something like
```

Comment:
How about putting `option(ENABLE_BIGINT "Enable wasm Bigint" OFF)` and putting this behind the option to make it easy to enable this?

Reply:
+1, and also can you elaborate on the comment a little more? It's not DWARF support doesn't exist in Binaryen at all, but it doesn't fully preserve some info, so...

Reply:
Sounds good, option added + comments.